speech/api/speech_async_rest.py

```diff
@@ -57,8 +57,12 @@ def main(speech_file):
     service_request = service.speech().asyncrecognize(
         body={
             'config': {
-                'encoding': 'LINEAR16',
-                'sampleRate': 16000
+                # There are a bunch of config options you can specify. See
+                # https://cloud.google.com/speech/reference/rest/Shared.Types/RecognitionConfig  # noqa
```

Comment:
(Or better yet: submit this as an issue for the linter team.)

On Jul 22, 2016 6:35 PM, "Ace Nassri" <ace.nassri@gmail.com> wrote:

> Can you do it with a goo.gl URL maybe?
>
> On Jul 22, 2016 6:34 PM, "Jerjou" <notifications@github.com> wrote:
>
>> In speech/api/speech_async_rest.py
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_GoogleCloudPlatform_python-2Ddocs-2Dsamples_pull_429-23discussion-5Fr71964562&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=xrSN-UXXowwKcdDyhdMK-aLiGWW6m2c6-QCsb8cp7Us&m=gWZJdp8CpjC7IUZI64TW3Yk_7MBXVGdX1sVW9uRGls8&s=-zAeXQpxSwoyMJN8QU5M5ZkP9YHO9fBhtwBDUMPjOHU&e=>
>> :
>>
>> > @@ -57,8 +57,12 @@ def main(speech_file):
>> >      service_request = service.speech().asyncrecognize(
>> >          body={
>> >              'config': {
>> > -                'encoding': 'LINEAR16',
>> > -                'sampleRate': 16000
>> > +                # There are a bunch of config options you can specify. See
>> > +                # https://cloud.google.com/speech/reference/rest/Shared.Types/RecognitionConfig  # noqa
>>
>> It's 'cuz it's a long line - if I don't put the # noqa comment, the
>> linter will complain that the line exceeds 79 characters in length - which
>> normally we don't want, but you can't really break up a url in a comment..
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_GoogleCloudPlatform_python-2Ddocs-2Dsamples_pull_429_files_0b67963aa67d769a1921b211b5f5955cef216b8f-23r71964562&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=xrSN-UXXowwKcdDyhdMK-aLiGWW6m2c6-QCsb8cp7Us&m=gWZJdp8CpjC7IUZI64TW3Yk_7MBXVGdX1sVW9uRGls8&s=-pwO5UVtuIN366Ael3grSWcKSQ1BCp4fT-rkvt5YH3g&e=>,
>> or mute the thread
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ACM8xeMBvhIRalU5jlBYgzhwiKogMIB2ks5qYW-2DngaJpZM4JTLlG&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=xrSN-UXXowwKcdDyhdMK-aLiGWW6m2c6-QCsb8cp7Us&m=gWZJdp8CpjC7IUZI64TW3Yk_7MBXVGdX1sVW9uRGls8&s=vLe7nW2FJLjMxfDhXGTmnj1thqsUXK1PD9Nk8eEM1jg&e=>
>> .
>>
>